### PR TITLE
Add README for Jan

### PR DESCRIPTION
**Pull request by justinmohs**
There you go

### CODE REVIEW

**Comment by justinmohs** [comment]
ref #1

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# This is for analysing p+p results from SMASH.
+
+just run the shell script
```
Comment (stdnmr) [inline README.md]: Could you please make this a real sentence and mention which shell script.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# This is for analysing p+p results from SMASH.
```
Comment (stdnmr) [inline README.md]: What about: 
```suggestion
# pp Analysis

This is repository for analyzing p+p results from SMASH.
```

**Review by stdnmr — CHANGES_REQUESTED** [review]
Good to see progress here, but a few small changes are still necessary. Thanks.

**Comment by stdnmr** [comment]
Did the changes myself, although I could not figure out, which shell script one has to run precisely.

**Pull request merged**